File: surf-friends/src/main/java/dev/slne/surf/friends/config/PluginConfig.java
package dev.slne.surf.friends.config;

import dev.slne.surf.friends.SurfFriendsPlugin;

import org.bukkit.configuration.file.FileConfiguration;

public class PluginConfig {
  public static FileConfiguration config() {
    return SurfFriendsPlugin.getInstance().getConfig();
  }

  public static void createConfig() {
    SurfFriendsPlugin.getInstance().saveDefaultConfig();
  }
}

File: surf-friends/src/main/java/dev/slne/surf/friends/config/PluginConfig.kt
package dev.slne.surf.friends.config

import dev.slne.surf.friends.SurfFriendsPlugin
import org.bukkit.configuration.file.FileConfiguration

object PluginConfig {
    fun config(): FileConfiguration {
        return SurfFriendsPlugin.instance.getConfig()
    }

    fun createConfig() {
        SurfFriendsPlugin.instance.saveDefaultConfig()
    }
}
